ui/src/ducks/app/monitoring.js

```diff
@@ -20,17 +22,18 @@ const defaultState = {
     list: [],
     error: null,
     isLoading: false,
-    isRefreshing: false
+    isRefreshing: false,
   },
   cluster: {
     apiServerStatus: 0,
     kubeSchedulerStatus: 0,
     kubeControllerManagerStatus: 0,
     error: null,
+    podStatusMessage: null,
```

Comment:
you should rename it to "prometheusPodStatus" instead to be clearer. and It should contain the status of the pod instead of only the message

Reply:
I will change the name from `podStatusMessage` to `prometheusPodStatusMessage`
Not sure if it's useful to store the status of the pod as in this case it's `pending`

Reply:
IMO, this kind of information would make more sense in the `monitoring` Redux state tree, not in `monitoring.cluster`. But we can move this to a debt ticket, as it's not critical.